Description: align create_memmap_backed_data() by default
 This avoids a bus error on armhf and should avoid
 slow downs on other architectures, thanks Seth Arnold
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/scikit-learn/+bug/1991621
Author: Graham Inggs <ginggs@debian.org>
Last-Update: 2022-10-08

--- a/sklearn/utils/_testing.py
+++ b/sklearn/utils/_testing.py
@@ -636,7 +636,7 @@
     )
 
 
-def create_memmap_backed_data(data, mmap_mode="r", return_folder=False, aligned=False):
+def create_memmap_backed_data(data, mmap_mode="r", return_folder=False, aligned=True):
     """
     Parameters
     ----------
